### PR TITLE
Use CsCodeGenerator for other parts of the source generator

### PR DESCRIPTION
**Pull request by kant2002**
- All generation is driven by CsCodeGenerator right now
- EventSystem generation based on assembly name
- Network serialized implemented

### CODE REVIEW

**Comment by kant2002** [comment]
After this PR lands, if something now working, I would like to know what patterns are not supported and I would quickly fix specific scenarios, unlikely and only EventSystem is potential candidate,

**Comment by libra34567** [comment]
tested in real project, steady, 
also works in build.

File: LittleToySourceGenerator/LittleToySourceGenerator.cs [hunk 1]
```diff
@@ -587,352 +622,385 @@ private static string GetDotsnetTypeName(ITypeSymbol typeSymbol)
...
 
-            if (hasRemovedEventViewTypes)
-            {
-                builder.AppendLine();
-                GenerateNotifyRemoveJob();
-            }
+        if (!ifDirty && !ifAdd && !ifRemove)
+        {
+            //return null;
+        }
```
Comment (libra34567) [inline LittleToySourceGenerator/LittleToySourceGenerator.cs]: what is the purpose of those three lines pls?

Comment (kant2002) [reply LittleToySourceGenerator/LittleToySourceGenerator.cs]: Ooh, that one. I should uncomment it. Thanks for spotting it.

**Review by libra34567 — APPROVED** [review]

**Pull request merged**